manage.py

```diff
@@ -0,0 +1,266 @@
+        kwargs.setdefault("capture_output", not verbose)
+        kwargs.setdefault("check", True)
+        kwargs.setdefault("env", {})
+        kwargs["env"].setdefault("PATH", os.environ["PATH"])
+        return run(["docker-compose", *compose_file_args, *args], **kwargs)
+
+    ctx.obj = dict()
+    ctx.obj["compose_cmd"] = _compose_cmd
+    ctx.obj["verbose"] = verbose
+    ctx.obj["yes"] = yes
+
+
+@cli.command()
+@click.pass_context
+def show_config(ctx):
+    """Show the merged docker-compose config."""
+    _docker_compose = ctx.obj["compose_cmd"]
+    click.echo(_docker_compose(["config"], capture_output=True).stdout)
+
+
+@cli.command()
+@click.option(
+    "--home-dir",
+    type=click.Path(),
+    # default="aiidalab-home",
+    help="Specify a path to a directory on a host system that is to be mounted "
+    "as the home directory on the AiiDAlab service. Uses docker volume if not provided.",
+)
+@click.option(
+    "--port",
+    # default=8888,
+    help="Port on which AiiDAlab can be accessed.",
+    show_default=True,
+)
+@click.option(
+    "--jupyter-token",
+    help="A secret token that is needed to access AiiDAlab for the first time. "
+    "Defaults to a random string if not provided (recommended).",
+)
+@click.option(
+    "--app",
+    multiple=True,
+    help="Specify app to install on first server start, using the same syntax as "
+    "`aiidalab install`. This option can be used multiple times to specify "
+    "multiple default apps.",
+)
+@click.option(
+    "--env-file",
+    type=click.Path(dir_okay=False, writable=True, path_type=Path),
+    default=".env",
+    help="The path of the env-file to use for configuration.",
+    show_default=True,
+)
+@click.pass_context
+def configure(ctx, home_dir, port, jupyter_token, app, env_file):
+    """Configure the local AiiDAlab environment."""
+    # First, specify the defaults.
+    env = OrderedDict(
+        [
+            ("AIIDALAB_HOME", "aiidalab-home"),
+            ("AIIDALAB_PORT", "8888"),
+            ("AIIDALAB_DEFAULT_APPS", "aiidalab-widgets-base"),
+            ("JUPYTER_TOKEN", token_hex(32)),
+        ]
+    )
+
+    # Next, update them with the currently stored values.
+    env.update(dotenv_values(env_file))
+
+    # Finally, update them with any of the values provided as arguments.
+    provided = {
+        "AIIDALAB_HOME": str(home_dir) if home_dir else None,
+        "AIIDALAB_PORT": str(port) if port else None,
+        "AIIDALAB_DEFAULT_APPS": " ".join(app),
+    }
+    env.update({key: value for key, value in provided.items() if value})
+
+    # Write environment to the env_file.
+    env_file.write_text(
+        "\n".join(f"{key}={value}" for key, value in env.items()) + "\n"
+    )
+    click.echo(f"Written configuration to '{env_file}'.")
+
+
+@cli.command()
+@click.option(
+    "--restart", is_flag=True, help="Restart AiiDAlab in case that it is already up."
+)
+@click.pass_context
+def up(ctx, restart):
```

Comment:
Can you add an option to use a different `.env` file? This would allow starting multiple containers at the same time, I guess.